packages/headless/src/controllers/commerce/search/summary/headless-search-summary.test.ts

```diff
@@ -0,0 +1,146 @@
+import {buildMockCommerceState} from '../../../../test/mock-commerce-state';
```

Comment:
The same comments I made in packages/headless/src/controllers/commerce/product-listing/summary/headless-product-listing-summary.test.ts apply here